### PR TITLE
Run and test webapp in CI

### PR DESCRIPTION
**Pull request by mateuszmidor**
Build, run and test the webapp from docker container
Based on https://github.com/kentik/sla_dashboard_webapp/pull/45

### CODE REVIEW

**Review by Fenthick — APPROVED** [review]

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            sla_dashboard:latest
+
+      - name: Warm-up time
+        run: sleep 5 
+
+      - name: Inspect webapp
+        run: docker logs sla_dashboard
+
+      - name: Test webapp 
+        run: curl 127.0.0.1:8050
```
Comment (Fenthick) [inline .github/workflows/test.yml:41]: curl will not fail when it gets HTTP error, e.g. 404. This fixes the problem:

```suggestion
        run: curl --fail --include 127.0.0.1:8050
```

**Review by Fenthick — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (mmac-m3a) [inline .github/workflows/test.yml:41]: Can we test both views (matrix and chart)?

Comment (mateuszmidor) [reply .github/workflows/test.yml:41]: > Can we test both views (matrix and chart)?

Sure we can.

**Comment by mateuszmidor** [comment]
This change is included in https://github.com/kentik/sla_dashboard_webapp/pull/47

**Pull request closed without merging**